testing/scenario_app/ios/Scenarios/ScenariosUITests/LocalizationInitializationTest.m

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "LocalizationInitialization.h"
+
+FLUTTER_ASSERT_ARC
+
+@implementation LocalizationInitializationTest
+
+- (void)setUp {
+  [super setUp];
+  self.continueAfterFailure = NO;
+
+  self.application = [[XCUIApplication alloc] init];
+  self.application.launchArguments = @[ @"--locale-initialization" ];
+  [self.application launch];
+}
+
+- (void)testNoLocalePrepend {
+  NSTimeInterval timeout = 10.0;
+
+  XCUIElement* textInputSemanticsObject =
```

Comment:
I know the engine has variable usage of dot notation, but the "modern" syntax (like circa 2006 modern) of properties would be:
```objective-c
XCUIElement* textInputSemanticsObject = [self.application.textFields matchingIdentifier:@"[en]"].element;
```